packages/backend/src/open_payments/quote/service.ts

```diff
@@ -229,10 +239,10 @@ function calculateFixedSendQuoteAmounts(
   quote: Quote,
   maxReceiveAmountValue: bigint
 ): CalculateQuoteAmountsWithFeesResult {
+  // TODO: derive fee from debitAmount and convert that to receiveAmount
```

Comment:
Took a shot at addressing it and was seeing some off-by-1 errors in the tests. Played with floor/ceiling differently in a few places with no differences. Might be worth tracing that a little further. If not would probably be better to turn the todo into an issue - not sure how critical you think this is @mkurapov

Reply:
Yes, I think there is some discrepancy here, or the current behaviour has some incorrect assumptions. Going to think it over some more... this is a tricky one